### ISSUE TITLE
pytorch的train2里package这个函数未定义

### ISSUE THREAD

**Issue by VectorChanger0** [Lapis-Hong/atec-nlp#3]
line 88:
data, targets = package(data_val[i:min(len(data_val), i+args.batch_size)], volatile=True)
未定义package函数

**Comment by Lapis-Hong** [comment]
yeah... pytorch version you need modify some thing to work, i mainly spend my time on tf version.
